tests/src/quickgui/app/main.qml

```diff
@@ -18,23 +18,56 @@ import QgsQuick 0.1 as QgsQuick
 import "."
 
 ApplicationWindow {
-  id: window
-  visible: true
-  visibility: "Maximized"
-  title: qsTr("QGIS Quick Test App")
+    id: window
```

Comment:
the version on master is better

Setup your Qt Creator to have 2 spaces for QML files similarly to formatting c++ files